jwst/straylight/straylight_step.py

```diff
@@ -14,7 +14,7 @@ class StraylightStep (Step):
 
     spec = """
          method = option('Nearest','ModShepard',default='ModShepard') #Algorithm method
-         roi = float(default = 50.0) # Region of interest
+         roi = integer(default = 50) # Region of interest
```

Comment:
Does `roi` need to be an integer?  In line 285 of straylight.py, you divide `roi` by 2.0, which makes `half_roi` a float.  Shouldn't `roi` be a float as well?

Reply:
I think routine shepard_2d_kernel is assuming roi is an integer.  
I program crashed when it was an odd vlalue.
I put in checks to make sure the roi is given as an even integer > 2 because that is what
the shepard_2d_kernel is assuming. 
I have change half_roi to be a integer - since it really should be since it is used in
xk, yk = np.meshgrid(np.arange(-half_roi, half_roi + 1),
                         np.arange(-half_roi, half_roi + 1))